templates/variablesDoc/customProperties.html.template

```diff
@@ -147,42 +231,56 @@
 
 			newObj = acc;
 			return newObj;
-		}, Object.create(null));
-
-		Object.keys(categorizedProperties).map((category) => { %>
-			<section class="section">
-				<div class="bounds"><div class="flex">
-					<div><h2 class="text--sectionTitle"><%= category %></h2></div>
-					<div class="tableWrapper"><table>
-						<thead>
-							<tr>
-								<th>Property name</th>
-								<th>Value</th>
-							</tr>
-						</thead>
-						<tbody>
-							<% _.each(categorizedProperties[category], (property) => {
-								const { name, value } = property; %>
+		}, Object.create(null)); %>
```

Comment:
Not a blocker for this PR, but noticed a small thing with `categorizedProperties`...

1) the initial value for this `reduce` can be an object literal.
2) there's no need to declare `newObj` outside of the function scope of `reduce`.

The value of `acc` is:
> the accumulated value previously returned in the last invocation of the callback, or `initialValue`, if supplied.

In other words, each iteration of `reduce` will pass the previously returned value to `acc`. If it's the first iteration, `initialValue` will be used (`Object.create(null)` or `{}`).